src/Module/Itemsource.php

```diff
@@ -17,10 +17,18 @@ public static function content()
 			return;
 		}
 
+		$a = self::getApp();
+
+		if (!empty($a->argv[1])) {
+			$guid = $a->argv[1];
+		}
+
+		$guid = defaults($_REQUEST['guid'], $guid);
```

Comment:
So `$_REQUEST` is more important than the arguments? Because there is no check if `$guid` is already set.

Reply:
Yep.